cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -2521,6 +2521,45 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 			});
 	});
 
+	it('Links through to "What did you think of this service? "', function () {
+		cy.enterFirstNameLastNameandEmail(
+			this.testData.firstName,
+			this.testData.lastName,
+			this.testData.email
+		);
+		cy.returningToTeaching(false);
```

Comment:
I think a lot of this setup can be skipped - we test this in other places already a number of times... I get it may be less of a user acceptance test, but would it be better/more maintainable to just load the completion page and check the link works?

Reply:
Yaah, completely agree. I'll change it.